.clang-tidy

```diff
@@ -1,15 +1,27 @@
 ---
 # Note on checks are disabled on purpose
 #
+# - abseil-cleanup-ctad
+#   Requires C++17 and higher.
+#
 # - abseil-no-namespace
 #   https://bugs.llvm.org/show_bug.cgi?id=47947
 #
+# - bugprone-exception-escape
+#   https://github.com/llvm/llvm-project/issues/54668 (seems to be fixed in LLVM17)
+#
 # - bugprone-reserved-identifier
 #   Some macros need to be defined for portability purpose; e.g. _BSD_SOURCE.
 #
 # - modernize-redundant-void-arg
 #   Some source should be strictly C99 and func(void) should be used.
 #
+# - google-readability-casting
+#   https://github.com/llvm/llvm-project/issues/57959
+#
+# - readability-redundant-string-cstr
+#   This is good to have, but needs a large refactor across the codebase.
```

Comment:
Please move this comment down below in the secion for future work.

Reply:
Done.